os/board/rtl8721csm/src/component/common/api/wifi/wifi_ind.c

```diff
@@ -197,9 +197,15 @@ void wifi_indication( rtw_event_indicate_t event, char *buf, int buf_len, int fl
 #if(WIFI_INDICATE_MSG==1)
 			printf("\n\r%s(): WIFI_EVENT_ICV_ERROR\n", __func__);
 #endif
+			break;
 		case WIFI_EVENT_CHALLENGE_FAIL:
 #if(WIFI_INDICATE_MSG==1)
 			printf("\n\r%s(): WIFI_EVENT_CHALLENGE_FAIL\n", __func__);
+#endif
+			break;
+		case WIFI_EVENT_STA_FOURWAY_HANDSHAKE_DONE:
+#if(WIFI_INDICATE_MSG==1)
+			printf("\n\r%s(): WIFI_EVENT_STA_FOURWAY_HANDSHAKE_DONE\n", __func__);

```

Comment:
it's not permitted to use printf in driver.
you can use vdvdbg for information vddbg for logging error.